tools/replay/util.cc

```diff
@@ -317,7 +317,24 @@ std::string decompressBZ2(const std::byte *in, size_t in_size, std::atomic<bool>
   return {};
 }
 
-void precise_nano_sleep(int64_t nanoseconds) {
+void precise_nano_sleep(long nanoseconds) {
```

Comment:
```suggestion
void precise_nano_sleep(int64_t nanoseconds) {
```

This doesn't match the definition in the header file, and doesn't compile on MacOS.